.github/workflows/CI.yml

```diff
@@ -34,6 +34,7 @@ jobs:
       - uses: julia-actions/setup-julia@v2
         with:
           version: ${{ matrix.version }}
+          include-all-prereleases: true
```

Comment:
It seems this implies that currently only the prerelease is tested but no stable release anymore...